src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/CommunityAPIClient.kt

```diff
@@ -1,65 +1,26 @@
 package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.component
 
-import mu.KLogging
-import net.logstash.logback.argument.StructuredArguments
 import org.springframework.stereotype.Component
 import org.springframework.web.reactive.function.client.WebClient
-import org.springframework.web.reactive.function.client.WebClientResponseException.BadRequest
 import reactor.core.publisher.Mono
 
 @Component
 class CommunityAPIClient(
   private val communityApiWebClient: WebClient,
 ) {
-  companion object : KLogging()
-
-  fun makeAsyncPostRequest(uri: String, requestBody: Any) {
+  fun makeFireAndForgetRequest(uri: String, requestBody: Any) {
     communityApiWebClient.post().uri(uri)
       .body(Mono.just(requestBody), requestBody::class.java)
       .retrieve()
       .bodyToMono(Unit::class.java)
-      .onErrorResume { e ->
-        handleResponse(e, requestBody)
-        Mono.empty()
-      }
-      .subscribe()
-  }
-
-  fun makeAsyncPatchRequest(uri: String, requestBody: Any) {
-    communityApiWebClient.patch().uri(uri)
-      .body(Mono.just(requestBody), requestBody::class.java)
-      .retrieve()
-      .bodyToMono(Unit::class.java)
-      .onErrorResume { e ->
-        handleResponse(e, requestBody)
-        Mono.empty()
-      }
-      .subscribe()
+      .block()
   }
 
   fun <T : Any> makeSyncPostRequest(uri: String, requestBody: Any, responseBodyClass: Class<T>): T {
     return communityApiWebClient.post().uri(uri)
       .body(Mono.just(requestBody), requestBody::class.java)
       .retrieve()
       .bodyToMono(responseBodyClass)
-      .onErrorMap { e ->
-        handleResponse(e, requestBody)
-        e
-      }
-      .block()
-  }
-
-  fun handleResponse(e: Throwable, requestBody: Any): String {
-    val responseBodyAsString = when (e) {
-      is BadRequest -> e.responseBodyAsString
-      else -> e.localizedMessage
-    }
-    logger.error(
-      "Call to community api failed",
-      e,
-      StructuredArguments.kv("req.body", requestBody),
-      StructuredArguments.kv("res.body", responseBodyAsString)
-    )
-    return responseBodyAsString.let { it } ?: "No response body message"
+      .block()!!
```

Comment:
Try verify the assumption that we can rely on Spring+Sentry to provide enough and good context for these kind of situations, run this locally with Sentry to have supporting data